bot/exts/halloween/hacktoberstats.py

```diff
@@ -434,12 +434,13 @@ async def _categorize_prs(prs: List[dict]) -> tuple:
         'hacktoberfest-accepted.
         """
         now = datetime.now()
+        oct3 = datetime(CURRENT_YEAR, 10, 3, 23, 59, 59, tzinfo=None)
         in_review = []
         accepted = []
         for pr in prs:
             if (pr['created_at'] + timedelta(REVIEW_DAYS)) > now:
                 in_review.append(pr)
-            elif await HacktoberStats._is_accepted(pr):
+            elif (pr['created_at'] <= oct3) or await HacktoberStats._is_accepted(pr):
```

Comment:
Doesn't this add all PRs from before October 3rd to the list of accepted ones? Shouldn't `and` be used instead of `or`?

Reply:
👋 All PRs created Oct 1-3rd on GitHub count toward Hacktoberfest, irrespective of if they were merged etc.

We updated the rules on Oct 3rd, requiring that any PRs past that point are merged/approved/labelled as `hacktoberfest-accepted` :)